exercises/clock/canonical-data.json

```diff
@@ -291,195 +361,225 @@
         {
           "description": "clocks with same time",
           "property": "equal",
-          "clock1": {
-            "hour": 15,
-            "minute": 37
-          },
-          "clock2": {
-            "hour": 15,
-            "minute": 37
+          "input": {
+            "clock1": {
```

Comment:
For these equality cases I was not sure if each clock should have its own "input" object or if the policy only pertains to the test case directly.  I will add input objects for each of the two clocks if the consensus is to do so.

Reply:
I think the way you've set it up currently is fine.